chromium/preloadlist/preloadlist.go

```diff
@@ -21,6 +21,24 @@ const (
 	ForceHTTPS = "force-https"
 )
 
+// HstsPreloadStatus indicates if a domain is preloaded.
+//
+// A domain can be preloaded by virtue of itself being on the preload list,
+// or by having one of its ancestor domains on the list and having
+// "include_subdomains" set to true on that ancestor domain.
+type HstsPreloadStatus int
+
+const (
+	// NotFound indicates that domain not preloaded.
+	NotFound HstsPreloadStatus = iota
```

Comment:
Nit: `EntryNotFound` (to prevent it from being confused with something like an HTTP 404 error).